content/docs/release-notes/2305-release-notes.md

```diff
@@ -0,0 +1,187 @@
+	* The paint brush workflow also includes runtime scripting support for real-time modifications of the terrain using a painting API.  
+	* Generalized Paint Brush documentation: https://www.o3de.org/docs/user-guide/components/reference/paintbrush/paintbrush/  
+	* Image Gradient Component with Paint Brush support: https://www.o3de.org/docs/user-guide/components/reference/gradients/image-gradient/  
+	* Terrain Macro Material Component with Paint Brush support: https://www.o3de.org/docs/user-guide/components/reference/terrain/terrain-macro-material/  
+* Added the Terrain Developer Guide that explains how developers can use and extend the terrain system: https://www.o3de.org/docs/user-guide/visualization/environments/terrain/terrain-developer-guide/
+* Sphere and disk lights now support shadow caching, which is accessible as a checkbox on the light component. This makes it so shadows do not re-render when nothing in their field of view has changed, and can greatly accelerate shadows on mostly static scenes. Models which have vertex animation in the shader can toggle on "Always moving" in the mesh component to ensure that they will always trigger updates when in the view of a cached shadow.
+* Added support for stereoscopic rendering (such as VR support) via OpenXR within O3de. This support was added through the Vulkan backend and it has been extensively tested using Quest 2. As part of this effort we are able to support Link mode where the rendering is done on PC and can be viewed on a stereoscopic display or the app can be run on the device natively. A special optimized VR pipeline has been added to Atom to help support running the app natively on the device.
+* The texture streaming support has been expanded to Vulkan backend. Now we have image streaming control which can load or evict texture mips automatically at runtime based on the streaming image pool's budget.
+* The shader library, material system, and builders have been updated to support a layer of abstraction between lighting and materials, called the material pipeline. It uses new builders and new features from the asset system to build unique shaders for each render pipeline. Material types can provide custom data structures and evaluator functions for vertex, pixel, geometry, and lighting stages. The builder inserts any of the custom include files with overridden data types and functions as it builds all of the combinations of shaders listed in the material pipeline.  
+	* The result of this is a concrete material type referencing all of the different combinations of shaders for each pipeline, with the customization stitched in. All of this is saved to the intermediate assets folder and processed by the AP to generate Final product material types and shaders. The material pipeline also includes a Lua script that will be invoked whatever material properties or the active render pipeline changes to enable or disable shaders based on the current state.  
```

Comment:
"processed by the AP" -> "processed by the Asset Processor"?  Not sure if it's called AP throughout documentation, but figured I'd call it out here to make sure.